lib/EntryManager/EntrySelector.js

```diff
@@ -92,6 +92,41 @@ class EntrySelector extends React.Component {
     }
   }
 
+  getActionMenu = item => ({ onToggle }) => {
+    const { clonable, onClone } = this.props;
+    const actions = [];
+
+    // Is cloanable
+    if (clonable) {
+      actions.push({
+        label: <FormattedMessage id="stripes-components.button.duplicate" />,
+        id: 'clickable-copy-item',
+        icon: 'duplicate',
+        onClick: () => {
+          onClone(item);
+          onToggle();
+        },
+      });
+    }
+
+    // No actions to show
+    if (!actions.length) {
+      return null;
+    }
+
+    return actions.map(button => (
+      <Button
+        buttonStyle="dropdownItem"
+        onClick={button.onClick}
+        id={button.id}
+      >
+        <Icon icon={button.icon}>
+          {button.label}
+        </Icon>
+      </Button>
+    ));
+  }
+
```

Comment:
It looks like "Duplicate" is the only option ever available - if that's the case, we could just return a `<Button>` for that, and don't have to map over `actions`. That would cut `getActionMenu()` in half.